docs/decisions/template.md

```diff
@@ -0,0 +1,80 @@
+---
+# These are optional elements. Feel free to remove any of them.
+status: {proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)}
```

Comment:
I would avoid both `proposed` and `rejected`, since we'll typically discuss the decisions in a PR.  Decisions should be PRd with `accepted` state, which will become true when merged, but rejected PRs will just be closed.

Reply:
(I still think this field has value and should not be optional, because of the `deprecated` and `superseded` statuses)

Reply:
> I would remove both `proposed` and `rejected`, since we'll typically discuss the decisions in a PR. Decisions should be PRd with `accepted` state, which will become true when merged, but rejected PRs will just be closed.

I agree with you about`'proposal`. But I think it is interesting to keep track of `rejected` records. It is also interesting to remember why you do not something.